src/main/java/org/terracotta/offheapstore/OffHeapHashMap.java

```diff
@@ -1068,6 +1066,10 @@ protected boolean tryIncreaseReprobe() {
     }
   }
 
+  protected void conditionallyShrink() {
```

Comment:
This is going to look weird next to the other public shrink method as this is not the conditional variant of the shrink method.  Maybe we could call this shrinkTable?